### PR TITLE
Feature/homepage temp

### PR DESCRIPTION
**Pull request by kcimartinez**
Add homepage component using only local assets. Fetching data from Contentful to be refactored later.

note: branch is named homepage-temp since I had trouble rebasing to main.

https://audiophile-ecommerce-git-feature-homepage-temp-muneyy.vercel.app?_vercel_share=bwHhV1jgokV573YGtB6ZjB5w9nDf3MHE

### CODE REVIEW

File: src/components/Homepage/subcomponents/CategoriesSection.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+const CategoriesSection = () => {
+  return (
+    <section className={styles.sectionCategories}>
+      {CATEGORIES_LIST.map((category) => (
+        <div className={styles.categoryCard} key={category}>
+          <div className={styles.imageContainer}>
+            <Image
+              src={
+                category === 'Headphones'
```
Comment (eremocal) [inline src/components/Homepage/subcomponents/CategoriesSection.tsx]: you can refactor this to

```
type CategorySrc = {
  [key: string]: string;
};

const categorySrc: CategorySrc = {
  Headphones: categoryHeadphones.src,
  Speakers: categorySpeakers.src,
  Earphones: categoryEarphones.src,
};

const src = categorySrc[category];
```

File: src/components/Homepage/Homepage.tsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+import bgBannerDesktop from '@/assets/image-hero.jpg'
+import bgBannerMobile from '@/assets/image-header.jpg'
+import bgFeaturedTwoDesktop from '@/assets/zx7-desktop.jpg'
+import bgFeaturedTwoMobile from '@/assets/zx7-mobile.jpg'
+import BannerSection from './subcomponents/BannerSection'
+import CategoriesSection from './subcomponents/CategoriesSection'
+import FeaturedSection from './subcomponents/FeaturedSection'
+
+const Homepage = () => {
+  const dynamicBackgroundImagesStyle = () => {
```
Comment (eremocal) [inline src/components/Homepage/Homepage.tsx:11]: is it possible not to use inline-style?

Comment (kcimartinez) [reply src/components/Homepage/Homepage.tsx:11]: I've explored other ways to do this but this is the solution I arrived at where I did not need to install additional libraries such as styled-components to integrate JS imports to CSS.

File: src/components/Homepage/subcomponents/CategoriesSection.tsx (hunk 1 above)
Comment (kcimartinez) [reply src/components/Homepage/subcomponents/CategoriesSection.tsx]: Refactored. This is much cleaner than my code.

**Review by eremocal — COMMENTED** [review]
image
could you please check it seems like the image is blurry on desktop

**Comment by kcimartinez** [comment]
> image could you please check it seems like the image is blurry on desktop

It was indeed blurry. Fixed it by using a higher resolution image instead.

**Review by eremocal — COMMENTED** [review]
image
please align the text to the header

**Review by eremocal — APPROVED** [review]

**Pull request merged**